data/hai/hai reveal 1 intro.txt

```diff
@@ -11,665 +11,19 @@
-			`	He asks a few questions for clarity, and when you're done he requests you wait while he makes a call. He leaves the room for a long while, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that most of humanity's food styles originated here.`
-			`	Eventually Mr. Yang returns. "So sorry for that. I hope you were looked after suitably while you waited."`
-			`	Once you nod your assent, he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official "line" is that the government does not trade in rumors. However, an appropriate response is clearly needed. Republic Intelligence briefed the Cabinet of the Free Worlds Senate just yesterday. They have demanded that they have a diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you to meet two individuals known as Katya Reynolds and Samuel Remington on Deep and Foundry respectively. I believe you already know them?"`
-			choice
-				`	"Yes."`
-				`	"Samuel Remington?"`
-					goto sam
-			`	"Excellent," he says as he begins to gather stacks of papers, shoving them into a briefcase.`
-				goto end
-			label sam
-			`	"Remington is the Syndicate's head of public relations, and exactly the man we need in an event like this," he explains while gathering two datapads and a selection of folders, shoving them into a briefcase.`
-			label end
-			`	"Now, the situation is progressing quickly and plans are still being made, so I am not entirely sure if it is they themselves who will be coming with us. Someone of Remington's caliber is likely too busy to come personally, but we'll find out when we get there. In any case I think we should go to Deep first for speed, but you are the captain. Shall we be off?"`
-				accept
-	on visit
-		dialog phrase "generic stopover on visit"
-	on complete
-		log `Brought human diplomatic representatives from each human faction to Hai-home to address the crisis. Something drastic needs to be done to keep the secret from entirely bursting, but they know how long they will have to work with. Asked to rout pirates hoping to benefit from the chaos while they work on a solution.`
-
-event "pirates respond to leaks"
-	# Add pirate spawns to Waypoint and Heia Due
-	system "Waypoint"
-		government "Pirate"
-		add fleet "Large Northern Pirates" 3000
-		add fleet "Small Northern Pirates" 1000
-		add fleet "Large Core Pirates" 3000
-		add fleet "Small Core Pirates" 1000
-		remove fleet "Small Hai"
-		remove fleet "Large Hai"
-	system "Heia Due"
-		add fleet "Marauder fleet IX" 3000
-		add fleet "Large Northern Pirates" 3000
-		add fleet "Large Core Pirates" 3000
-		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
```

Comment:
The need to tweak fleet numbers aside, this event should probably not be entirely deleted at this point in time.